cumulusci/core/flowrunner.py

```diff
@@ -51,26 +51,34 @@
 
 """
 
-# we don't actually use this set of imports, they're just in type
-# comments, which require explicit runtime import when checking...
-try:
-    from typing import List
-except ImportError:  # pragma: no cover
-    pass
-
 import copy
 import logging
-from collections import defaultdict, namedtuple
+from collections import defaultdict
 from distutils.version import LooseVersion
 from operator import attrgetter
+from typing import (
+    Any,
+    DefaultDict,
+    Dict,
+    Optional,
+    Tuple,
+    Type,
+    TYPE_CHECKING,
+    List,
```

Comment:
Suggestions:
- `pre-commit install --install-hooks`
- `git commit --no-verify -m 'wip'` if you get annoyed.
- Also: https://marketplace.visualstudio.com/items?itemName=ms-python.isort